vignettes/s3-vector.Rmd

```diff
@@ -19,6 +19,11 @@ This vignette shows you how to create your own S3 vector classes. It focuses on
 
 I assume that you're already familiar with the basic machinery of S3, and the vocabulary I use in Advanced R: constructor, helper, and validator. If not, I recommend reading at least the first two sections of [the S3 chapter](https://adv-r.hadley.nz/s3.html) of _Advanced R_.
 
+This article refers to "vectors of numbers" as *double vectors*. Here, "double" stands for ["double precision floating point number"](https://en.wikipedia.org/wiki/Double-precision_floating-point_format). It is also the underlying *storage mode* for `numeric()` vectors in R, see the "Mode names" section in `?mode`. This is a deliberate choice, for two reasons:
```

Comment:
I'm not sure we need to define "storage mode" as if it's a general concept to be aware of? Also isn't the storage mode "numeric" rather than "double"? Maybe make it clear that we generally prefer the `typeof()` than the `mode()` which is a legacy of S? The challenge is to stay clear and to the point so it doesn't feel like a digression right at the start of the vignette, and also not to introduce further confusion with these tricky concepts.